config/environments/development.rb

```diff
@@ -42,4 +42,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      access_key_id: "AKIAIXRHXAZSSNZC4JVQ",
+      bucket: "grants.hifinfo.org",
+      secret_access_key: "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",
```

Comment:
Also, these are the same credentials in use on the production site.  @joedean Can you delete this access key, generate a new one, and update the ENV vars on heroku with the new ones?  Pushing this commit may have just given the public unlimited access to our S3 account, or possibly even to other AWS services as well.